### ISSUE TITLE
Validation for unique emails

### ISSUE THREAD

**Issue by stephensanwo** [supertokens/supertokens-python#264]
Currently using the Python SDK and React SDK, I might be missing something, but I noticed that the signup form does not correctly validate that an email has already been created (unique email). The documentation says that checking unique emails is part of the signup flow, but the signup still goes through and doesn't return a form field error when the email is not unique.
I know I can use the doesEmailExist function to check if the email is unique, and return this error, but I'll rather the backend handles this.

See below some snapshots

I have created a dummy user, as shown in the supertokens dashboard

<img width="1048" alt="image" src="https://user-images.githubusercontent.com/49814040/206898366-c60dcbad-f1a6-4122-989b-50b7bbb2d441.png">

However the code below does not return a form field error for email, showing that the user already exists. Instead it redirects to the /home, although it doesn't create a new user on the backend. I believe the flow should return a unique email error, without having to explicitly call the doesEmailExist function?

```js
import { signUp } from "supertokens-web-js/recipe/emailpassword";

async function signUpClicked() {
    try {
        let response = await signUp({
            formFields: [{
                id: "email",
                value: "test.user@email.com"
            }, {
                id: "password",
                value: "password123456"
            }]
        })

        if (response.status === "FIELD_ERROR") {
            response.formFields.forEach(formField => {
                if (formField.id === "email") {
                    window.alert(formField.error)
                } else if (formField.id === "password") {

                    window.alert(formField.error)
                }
            })
        } else {
            window.location.href = "/home"
        }
    } catch (err: any) {
        if (err.isSuperTokensGeneralError === true) {
            window.alert(err.message);
        } else {
            window.alert("Oops! Something went wrong.");
        }
    }
}
```

**Comment by rishabhpoddar** [comment]
Hey @stephensanwo thanks for opening this issue. @KShivendu will check it out tomorrow.

**Comment by rishabhpoddar** [comment]
This has been fixed and released in python SDK version >= 0.11.10
